.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -0,0 +1,367 @@
+    jobs:
+      - job: apachebench_test
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                echo "Deploy apachebench pod and run test"
+                cd hack/manifests
+                kubectl apply -f apache.yaml
+                echo "wait for pod to become ready"
+                kubectl rollout status deployment apachebench --timeout=30s
+                kubectl get pod -owide
+                mkdir apachebench
+                cd apachebench
+                AB_POD=$(kubectl get pod -l app=apachebench | grep apachebench | awk '{print $1}')
+                kubectl exec -it $AB_POD -- ab -n ${AB_REQUESTS} -c ${AB_CONCURRENCY} -r http://real-svc-00001.scale-test/ >> "ab_${AB_REQUESTS}requests_${NUM_REAL_REPLICAS_LB}kpods.log"
+                echo "collect cpu and memory usage after apachebench tests"
+                cd ..
+                mkdir after_lb_cpu_and_mem
+                cd after_lb_cpu_and_mem
+                echo "running k top node" 
+                kubectl top node >> "node_after_lb_tests.log"
+                echo "running k top pod"
+                kubectl top pod -A | grep cilium >> "pod_after_lb_tests.log"
+                cd ..
+                echo "Logs will be available as a build artifact"
+                ARTIFACT_DIR2=$(Build.ArtifactStagingDirectory)/after_lb_cpu_and_mem/
+                ARTIFACT_DIR=$(Build.ArtifactStagingDirectory)/apachebench/
+                echo $ARTIFACT_DIR
+                echo $ARTIFACT_DIR2
+                sudo rm -rf $ARTIFACT_DIR $ARTIFACT_DIR2
+                sudo mkdir $ARTIFACT_DIR
+                sudo mkdir $ARTIFACT_DIR2
+                sudo cp ./apachebench/* $ARTIFACT_DIR
+                sudo cp ./after_lb_cpu_and_mem/* $ARTIFACT_DIR2
+            name: "TestLBServices"
+            displayName: "Apachebench testing"
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: apachebench
+              pathtoPublish: "$(Build.ArtifactStagingDirectory)/apachebench"
+            condition: always()
+            name: "PublishResults"
+            displayName: "Apachebench Result Artifacts"
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: after_lb_cpu_and_mem
+              pathtoPublish: "$(Build.ArtifactStagingDirectory)/after_lb_cpu_and_mem"
+            condition: always()
+            name: "PublishResults2"
+            displayName: "Result Artifacts"
+  - stage: netperf_tests
+    displayName: "Run netperf tests"
+    jobs:
+      - job: netperf
+        timeoutInMinutes: 120
```

Comment:
What is the expected runtime?

Reply:
this test completes quickly, so I removed the timeout here